mlflow/entities/multipart_upload.py

```diff
@@ -0,0 +1,46 @@
+from dataclasses import dataclass
+from typing import List, Optional, Dict, Any
+
+from mlflow.protos.mlflow_artifacts_pb2 import (
+    CreateMultipartUpload as ProtoCreateMultipartUpload,
+    MultipartUploadCredential as ProtoMultipartUploadCredential,
+)
+
+
+@dataclass
+class MultipartUploadPart:
+    part_number: int
+    etag: str
+
+    @classmethod
+    def from_proto(cls, proto):
+        return cls(
+            proto.part_number,
+            proto.etag or None,
+        )
+
+
+@dataclass
+class MultipartUploadCredential:
+    url: str
+    part_number: int
+    headers: Dict[str, Any]
+
+    def to_proto(self):
+        credential = ProtoMultipartUploadCredential()
+        credential.url = self.url
+        credential.part_number = self.part_number
+        credential.headers.update(self.headers)
+        return credential
```

Comment:
/ai This data class represents a chunk in a multipart upload. `url` is a pre-signed URL and `headers` is headers used when uploading the chunk. How would you name this class?